torch/_subclasses/fake_tensor.py

```diff
@@ -381,6 +401,8 @@ def __torch_dispatch__(cls, func, types, args=(), kwargs=None):
                 return torch.device("meta")
             else:
                 return args[0].fake_device
+        elif func == torch.ops.aten.stride.default:
```

Comment:
just curious, why was this necessary/what does this accomplish ?

Reply:
TBH, I kind of have forgotten. I think it was a symbolic shapes / fake tensor interaction, maybe.

Reply:
I removed it and remembered. I'll push a comment